### ISSUE TITLE
"base" printing doesn't work on Python 3.7

### ISSUE THREAD

**Issue by YtvwlD** [chaosdorf/labello#9]
The resulting error message is not very helpful:

```
172.17.0.1 - - [22/Nov/2018 19:21:00] "POST /base HTTP/1.1" 301 -
multipart/form-data {'boundary': '---------------------------94047808817135404431800418516'}
<_io.BufferedReader name=4>
ERROR: 'CONTENT-LENGTH'
```

**Comment by YtvwlD** [comment]
The bug is this: https://bugs.python.org/issue34226

**Comment by YtvwlD** [comment]
This has been fixed in 2019 by 7ef59dcda22f615e444273397f6b77f162809f46.
